contrib/opencensus-ext-azure/opencensus/ext/azure/trace_exporter/__init__.py

```diff
@@ -62,7 +63,7 @@ def __init__(self, **options):
         super(AzureExporter, self).__init__(**options)
         atexit.register(self._stop, self.options.grace_period)
 
-    def span_data_to_envelope(self, sd):
+    def span_data_to_envelopes(self, sd):
```

Comment:
Because "technically" this is exposed in the api, I would not want to change the name of this function. Making it plural also doesn't add TOO much value.